restore.go

```diff
@@ -143,6 +144,22 @@ func restoreContainer(context *cli.Context, spec *specs.LinuxSpec, config *confi
 	return handler.forward(process)
 }
 
+func handleContainerstatus(status libcontainer.Status, container libcontainer.Container, context *cli.Context) (int, error) {
+	switch status {
+	case libcontainer.Running:
+		return -1, fmt.Errorf("Container with id %s already running", context.GlobalString("id"))
+	case libcontainer.Destroyed:
+		defer func() {
```

Comment:
this defer will fire after `handleContainerstatus()` returns, is this intentional?  if so, why not just call `Destroy()` without the defer?